### ISSUE TITLE
Add JOSS to pubmed

### ISSUE THREAD

**Issue by pjotrp** [openjournals/joss#153]
Pubmed is a resource for biomedical publications. Getting a paper listed is critical for career development in bioinformatics. 

We have to find out what it takes to get a publication listed (volume and issue identifiers will help) on pubmed. Maybe we can submit papers individually or have them harvested on some tag/identifier as in https://github.com/openjournals/joss/issues/124. I'll try to submit GeneNetwork once we have those.

**Comment by tracykteal** [comment]
I agree it would be great to get articles indexed in PubMed and is important for bioinformatics in particular. It's unlikely the journal will get in through the MEDLINE journal selection process, since it's not biomedically focused. 

https://www.nlm.nih.gov/pubs/factsheets/j_sel_faq.html

However, it seems that if journal or paper content is put into PubMed Central they will be indexed in PubMed, so perhaps we could try that route?

http://www.ncbi.nlm.nih.gov/pmc/about/faq-pub/

There is a 'scientific quality standard' there that evaluates " whether the scientific and editorial character and quality of a journal merit its inclusion in PMC".

@pjotrp it would be great if there was a test paper you could try this with.

**Comment by genomematt** [comment]
The slight issue with the submitting directly to PubMed Central is that it requires the work having been supported by a PMC supporting funder if the Journal is not part of PMC.

**Comment by pjotrp** [comment]
The NIH is a funder of both PMC and GeneNetwork. So we should attempt this.

**Comment by konrad** [comment]
:+1: I would strongly support this suggestion. I have a piece of software that I would like to publish in JOSS but my collaboration partner would like to have it in a journal that is listed in Pubmed. Has there been any attempt to talk to NIH?

**Comment by kyleniemeyer** [comment]
We could try submitting for MEDLINE review—we do have plenty of biomedical-related articles: https://wwwcf.nlm.nih.gov/lstrc/lstrcform/med/index.html

**Comment by konrad** [comment]
Sounds good and I think this is likely to work. Who of JOSS would be in the position to submit this application? @arfon? Sorry for being pushy.

**Comment by kyleniemeyer** [comment]
I have been sort-of taking the lead on these indexing efforts, mainly with Google Scholar so far, so I can do this—perhaps tomorrow. Not sure how long it will take to get a response, though.

**Comment by konrad** [comment]
That's great! I also assume that the processing of the application might take a while. Many thanks and fingers crossed.

**Comment by HenrikBengtsson** [comment]
I noticed JOSS is listed on PubMed (https://www.ncbi.nlm.nih.gov/nlmcatalog/101708638) with

> In: PubMed: Selected citations only
> Current Indexing Status: Not currently indexed for MEDLINE. Citations are for articles where the manuscript was deposited in PubMed Central (PMC) in compliance with public access policies. For further information, see Author Manuscripts in PMC.

I also noticed that there is exactly one JOSS paper indexed there. Is this because JOSS just got added, or what caused that paper to get indexed and not others? Will all future JOSS paper be indexed automatically?

**Comment by arfon** [comment]
> Will all future JOSS paper be indexed automatically?

I'm afraid I don't know the answer to this question. PubMed is outside my area of expertise...

**Comment by konrad** [comment]
I hope this is okay for everybody but I am very interested in the proper indexing of JOSS in MEDLINE/PubMed. Due to that I sent the following request to the NCBI Help Desk (the PubMed Customer Service did not work for me).

> Dear NLM team,
> 
> since very recently the "Journal of open source software" (JOSS) [1] is indexed in MEDLINE/PubMed [2]. Currently, it seems as only one of its article was included to MEDLINE/PubMed (tested using this search [3]). JOSS is an rather open journal and the mentioned problem was raised in a GitHub issue. Could you please answer the following questions either via email to me or directly in the linked GitHub issue threat:
> 
> - Why is so far only one article indexed in MEDLINE/PubMed?
> - Will all future JOSS articles be indexed automatically in MEDLINE/PubMed?
> - Will also JOSS articles be indexed if they were published before JOSS was added to be indexed in MEDLINE/PubMed?
> 
> Many thanks in advance.
> 
> Best wishes
> 
> Konrad Förstner
> 
> PS: I wanted to submit this request directly via the PubMed Customer Service form but got always errors (e.g. Error ID 5527bcab-4ff7-47af-94d6-ec827bde41ae).
>
> [1] http://joss.theoj.org/
> [2] https://www.ncbi.nlm.nih.gov/nlmcatalog/101708638
> [3] https://www.ncbi.nlm.nih.gov/pubmed?term=%22J%20Open%20Source%20Softw%22%5BJournal%5D
> [4] https://github.com/openjournals/joss/issues/153#issuecomment-337460297

As soon as I get a response I will inform you here.

**Comment by Kevin-Mattheus-Moerman** [comment]
Thanks for doing this. We'll see how they reply. If you do not hear anything soon we might follow up with an inquiry directly from the JOSS editorial team (I'll discuss this) as this might have more momentum. Also feel free to ask us in the future to help draft/format messages like this.

**Comment by konrad** [comment]
The automatic email response says that they will answer within 4 business days. I will let you know once I have a proper response.

**Comment by konrad** [comment]
I received (already some days ago - sorry for the delay) the following response:

> The journal you reference, Journal of open source software" (JOSS) is not a                                                                                                                  
> title indexed for MEDLINE. Please see https://www.ncbi.nlm.nih.gov/nlmcatalog/101708638                                                                                                               
>                                                                                                                                                                                     
> I would also like to suggest a recording of a webinar we recently held                                                                                                                       
> discussing how journals are added to both PubMed as a MEDLINE journal: 
> https://www.nlm.nih.gov/bsd/disted/video/selection.html

**Comment by genomematt** [comment]
I have gone through the process of suggesting JOSS for SCOPUS.  The tracking nr. for this title suggestion is: 46B06F641F8C7575.  The time line for this is 6-12 months...

**Comment by arfon** [comment]
Excellent. Thanks @genomematt!

**Comment by konrad** [comment]
Many thanks, @genomematt!

**Comment by sebastiz** [comment]
If you want to submit manually to google scholar by the way then its pretty straightforward. Follow here:...http://blog.impactstory.org/make-google-scholar-better/

I still think its really important to have the pubmed addition especially given the explosion of bioinformatics and R..This would be a real deal-breaker for the use of the rOpenSci pipeline. Any further developments in this?

**Comment by federeghe** [comment]
Another interesting place is Clarivate Analytics, the company maintaining Web of Science. This is the submission form if someone wants to try (the submitter must be an editor): http://mjl.clarivate.com/journal-submission/form/

**Comment by arfon** [comment]
> Hi all. Is there an action plan to take Pubmed/JOSS issue forward? Is it just a case of someone submitting an application to Pubmed as per @konrad above . I'm happy to do this if needed.

@sebastiz - that would be great, thanks! Let me know if you need any help with this.

**Comment by sebastiz** [comment]
Hi @arfon

So the pubmed registration needs to be done by the editors it turns out. The page for submission is here:
https://www.ncbi.nlm.nih.gov/pmc/pub/addjournal/

Basically before submission the following needs to happen:

The following requirements must be met before making any kind of application to PMC:

The journal must have a properly registered ISSN. This means that there must be a confirmed record for the journal in the official Register of the ISSN International Centre.
The journal must be in scope for PMC, as described in the PMC Scope section of the PMC Policies page.
The journal must be able to provide NLM with immediate access to the content at a publisher or third-party site, as required by NLM's Electronic Resources policy.
For publishers with at least a two-year history of quality scholarly publishing in the life sciences, a minimum of 25 peer reviewed articles (e.g., original research or review articles, clinical case reports) must be published in final form before you apply.

Can one of the editors submit for ISSN and NLM and then the journal can be submitted

**Comment by arfon** [comment]
Thanks for digging into this @sebastiz. We already have an ISSN for JOSS. Unfortunately we don't currently doing the following:

> A journal must provide PMC with the full text of articles in an XML (eXtensible Markup Language) format that conforms to an acceptable journal article DTD (Document Type Definition) and meets the PMC Minimum Data Criteria. PMC does not accept articles in HTML format.

> NLM recommends that data be submitted in XML conforming to the NISO JATS Journal Publishing Tag Set, but PMC will also accept data in other full-text article DTDs that are widely used in life sciences journal publishing.

**Comment by sivaramambikasaran** [comment]
> I have gone through the process of suggesting JOSS for SCOPUS. The tracking nr. for this title suggestion is: 46B06F641F8C7575. The time line for this is 6-12 months...

Any progress on this? Also, will the already submitted publications also be considered scopus indexed. Sorry if the question is stupid, just that I am ignorant of the entire process.

**Comment by goerz** [comment]
I've had a PI shoot down the suggestion to publish in JOSS because it's not indexed by Web of Science. It would be really great if that could change.

**Comment by pjotrp** [comment]
@goerz that should not prevent you from publishing in JOSS. It makes your software citeable while you can still publish methods etc. in other journals. I have no doubt, btw, that JOSS will be indexed in time. With 500 publications JOSS is getting harder and harder to ignore. This is a true outlet of scientific work. Better than many other journals out there ;). Do disrupt with us. Web of Science will wake up to it.

**Comment by arfon** [comment]
> Any progress on this? Also, will the already submitted publications also be considered scopus indexed. Sorry if the question is stupid, just that I am ignorant of the entire process.

I don't know about PMC progress (I didn't submit the original request), but we're working on adding an ethics statement in #496 which was a blocker for us being indexed in Scopus.

I'm also actively investigating creating JATS XML for JOSS papers which would be necessary for us to be indexed in PMC.

**Comment by meg-simula** [comment]
[I found this thread when checking if JOSS is Scopus indexed.] What is the current status on this? I noticed that the ethics statement issue was resolved. Is there any remaining issues re: scopus indexing (i.e. anything I can contribute with)?

**Comment by arfon** [comment]
>  Is there any remaining issues re: scopus indexing (i.e. anything I can contribute with)?

Thanks for asking @meg-simula. At this point we're waiting to hear back from Scopus. I resubmitted JOSS for indexing about a month ago.

**Comment by jbloom** [comment]
> > Any progress on this? Also, will the already submitted publications also be considered scopus indexed. Sorry if the question is stupid, just that I am ignorant of the entire process.
> 
> I don't know about PMC progress (I didn't submit the original request), but we're working on adding an ethics statement in #496 which was a blocker for us being indexed in Scopus.
> 
> I'm also actively investigating creating JATS XML for JOSS papers which would be necessary for us to be indexed in PMC.

I was just curious if there is progress on getting JOSS papers indexed by PMC. We'd like to submit to JOSS, but having PMC indexing is important for NIH grants.

**Comment by iskandr** [comment]
Is there anything us observers can do to help get JOSS indexed by PubMed?

**Comment by arfon** [comment]
The main issue here is that PubMed needs JOSS to produce JATS XML before they will index us. Unfortunately the Pandoc--JATS conversion is somewhat buggy and I got stuck on this a while ago in https://github.com/openjournals/whedon/pull/45

I definitely want to get this done at some point, it's just hard to find the time to debug the conversion sorry 😞

**Comment by DominiqueMakowski** [comment]
Happy new year all ☺️ 

Just came here for some updates about the referencing of JOSS in various databases (unfortunately quite important for many researchers). If I understand the thread correctly:

- *PubMed*: blocker issue (https://github.com/openjournals/whedon/pull/45)
- *Scopus*: pending (submitted in September) - any news on that?
- *Web of Science*: not submitted (?)

**Comment by arfon** [comment]
Hi @DominiqueMakowski 

> - PubMed: blocker issue (openjournals/whedon#45)

Correct. Although we made a good bit of progress on this last week with https://github.com/jgm/pandoc/pull/6020

> - Scopus: pending (submitted in September) - any news on that?

I just asked for a status update from Scopus yesterday. Not sure of the status of this but it's still pending.

> - Web of Science: not submitted (?)

We submitted to Web of Science a few weeks ago but haven't heard anything back yet.

**Comment by DominiqueMakowski** [comment]
Thanks for everything @arfon 🙌

**Comment by dpole** [comment]
Hi @arfon , is there any update on Scopus/WOS? 
I have to decide which journal to target. JOSS looks perfect except for this databases issue, which is somewhat relevant to my academic system -- unfortunately :(  .
Thanks for your work!

**Comment by arfon** [comment]
I’m afraid not sorry. These systems are essential information ‘black boxes’ - we’ve asked to be indexed but we have no way of checking on the status of this request 😔

**Comment by genomematt** [comment]
Just got this in one of my old mailboxes. Its been quite a while since I suggested it
@arfon I have forwarded to your gmail

```Title: Journal of Open Source Software 
ISSN / E-ISSN: / 2475-9066 
Publisher: Open Journals 

Dear Dr. Matthew Wakefield, 

The title mentioned above has been evaluated for inclusion in Scopus by the Content Selection & Advisory Board (CSAB). The review of this title is now complete and the CSAB has advised to not accept the title for Scopus inclusion at the present time. For your information, the reviewer comments are copied below: 

I really like the concept of a journal publishing software development. Usually, software developers get insufficient credit for their work which takes significant effort. This negatively impacts on attracting top-level PhD students to work on significant software development and on subsequent tenure&promotion. Unfortunately though this journal does seem to accept most submissions. Reviewing is somewhat limited. So, accepting this into Scopus would not be consistent with other journals' acceptance. Rigorous reviewing of implementation work guaranteeing highest standards would benefit the entire area and allow us to accept such a title into Scopus. Unfortunately, the current title does not do this. 


If in the future these comments are addressed, you may decide to submit a new application at any time after the following date: March 2022. 
At that time, you will be required to upload a cover letter detailing how the above comments have been addressed. 

Finally, we strongly advise you to read through our FAQs: 
Helping to improve the Scopus submission & success process for editors and publishers: https://embed.widencdn.net/download/elsevier/olbvgdobzi/SC_FAQ-content-selection-process_Jan2019.pdf?u=simrzc 
Role of an editor: https://www.elsevier.com/__data/assets/pdf_file/0005/95117/SC_FAQ-Role-of-an-Editor-22092014.pdf 

Yours sincerely, 


Scopus Title Evaluation Support 
titlesuggestion@scopus.com
```

**Comment by genomematt** [comment]
Note that 12 March 2018 was the submission date https://github.com/openjournals/joss/issues/153#issuecomment-372170246

**Comment by arfon** [comment]
Thanks @genomematt - I have a bunch of issues with that response from Scopus but am at a bit of a loss how to proceed. 

I’ll sync up with the JOSS editorial team and try and figure out what next steps might look like here.

**Comment by tknopp** [comment]
@arfon I would encourage you to get in contact with Scopus and respond to their review. Question is whether you can derive some acceptance rate for JOSS. That would be helpful in the response.

**Comment by arfon** [comment]
> @arfon I would encourage you to get in contact with Scopus and respond to their review. Question is whether you can derive some acceptance rate for JOSS. That would be helpful in the response.

Thanks for the encouragement @tknopp. We will do this at some point and we've discussed how we can derive an acceptance rate based on the papers we have rejected as out of scope or not up to standard.

Do you have any contacts at Scopus? The reassessment every ~2 years is a little frustrating, especially when they seem to have misunderstood how JOSS functions.

**Comment by Oftatkofta** [comment]
Hi all, 

Has there been any new developments on this front, and is there anything an eager bystander can help with?

**Comment by tknopp** [comment]
I forgot to answer to @arfon. No, I do not have a more direct contact to scopus but I have successfully landed another journal in scopus (https://journal.iwmpi.org) and went through the registration process.

From my perspective the most important thing is to derive rejection / out of scope metrics. Once that is in place it should be relatively straight forward to write a good response to the initial review. The most important thing to think about is whether an open review model leads to the situation that more "low-quality" paper are accepted since nobody wants to raise critical points in public. This issue for open review models needs to be discussed and it needs convincing arguments and examples, how this issue is resolved. On the pro side one can then also list all the positive aspects of open review models but critical reflection on the issue raised by the scopus reviewer certainly raises the chances to get listed.

**Comment by danielskatz** [comment]
I think there's a fundamental problem here if one of the SCOPUS criteria is acceptance rate, given that our goal is to collaboratively improve submission to the point where they become acceptable, not to reject them.

**Comment by tknopp** [comment]
I fully agree with that. But still it would be interesting to see some numbers about the acceptance rate to discuss the issue raised by the reviewer. If its 100% acceptance rate one would answer differently to Scopus than if the acceptance rate is 50%.

**Comment by danielskatz** [comment]
I would guess it's around 80%, mostly based on some rejected before review (or in review) as being out of scope, typically insufficient work, and a very small number rejected because the changes needed are more than the authors want to do. @arfon and @csoneson - would either of you have a way to figure this out?

**Comment by csoneson** [comment]
I can find 137 pre-review issues and 12 review issues that were opened in 2020 and have the 'rejected' label attached. In total, 547 pre-review issues and 391 review issues were opened in 2020.

**Comment by tknopp** [comment]
By the way, I had a look in my mailbox and the mail latency for questions to titlesuggestion@scopus.com was quite good (1 day). Thus, it might be worth getting in touch with them how to proceed. More concretely one could ask if there is the possibility of writing a "rebuttal" letter when reproposing the journal for inclusion in scopus.

**Comment by arfon** [comment]
**On PubMed:**

> Has there been any new developments on this front, and is there anything an eager bystander can help with?

_Some_.  @tarleb and I are working on some updates to the document production toolchain for JOSS to product JATS output. Hopefully this should be complete in the next couple of months and this should allow us to provide these files to PubMed for indexing.

**On Scopus:**

> By the way, I had a look in my mailbox and the mail latency for questions to titlesuggestion@scopus.com was quite good (1 day). Thus, it might be worth getting in touch with them how to proceed. More concretely one could ask if there is the possibility of writing a "rebuttal" letter when reproposing the journal for inclusion in scopus.

This sounds like a good idea. I will try to find some time to work on this as it's an important topic for our authors but I confess a combination of lack of time and lack of affection for Scopus are dual challenges for me here...

**Comment by MikaelSellin** [comment]
Sounds great with the progress on the PubMed indexing front, @arfon. For us in the biomedical research community, this is really the one portal everyone uses to discover new science (and hopefully soon new science software). It has been a pleasure for us interacting with JOSS. But, I also notice that among fellow biomedical researchers, JOSS remains largely unknown, which seems like a shame considering the central role software development now has in our field. PubMed-indexing would go a long way towards amending this lack of visibility. Don´t take this as a stressor, simply some optimistic cheering on from the sideline. Thanks for all the hard and altruistic work by you and all the JOSS affiliates.

**Comment by tknopp** [comment]
I am also cheering from the sideline but its really like Mikael said. In order to get more attraction it is crucial to get into the indexing services. I love the idea and the excellent work behind JOSS but right now I would probably use another Journal in many cases since the downsides are too large at the moment. Although I really dislike the current system, as a matter of fact, citations are the currency in academic world (at least in Germany).

@arfon: I could offer getting in contact with Scopus and ask if it is possible to send a rebuttal letter if you want. Would put you in CC.

**Comment by kephale** [comment]
> _Some_. @tarleb and I are working on some updates to the document production toolchain for JOSS to product JATS output. Hopefully this should be complete in the next couple of months and this should allow us to provide these files to PubMed for indexing.

Just checking to see how this has been going. We've been preparing a submission for JOSS (https://arxiv.org/pdf/2110.00601.pdf), but PubMed indexing would be really useful for NIH purposes. 

A related question, will these updates mean that existing JOSS papers can be indexed by PubMed? e.g. could we submit anyway and assume that this will eventually be resolved in the future?

**Comment by tknopp** [comment]
I can only speak for Scopus and they do not index existing papers prior to acceptance into the index.

**Comment by tarleb** [comment]
Development of the JATS toolchain had to go on a hiatus for a few months, but now we are back at it. We currently have JATS files for 1332 of the 1443 published papers. The last 10% all require at least some manual fixes to the input files, e.g. to allow the proper conversion of math formulas. There is no detailed timeline, but we are making good progress.

**Comment by tarleb** [comment]
There is now PR #1022 that adds a JATS XML archive. The files all pass the JATS DTD validation at https://validator.jats4r.org.

**Comment by janfreyberg** [comment]
Does the completion of this mean Pubmed will start indexing JOSS?

Checking on the journal in the NLM catalogue indicates no, so far: https://www.ncbi.nlm.nih.gov/nlmcatalog/101708638

Will this be possible now?

**Comment by arfon** [comment]
> Will this be possible now?

Yes, it should be, although I've not managed to investigate how to get indexed thus far. Do you happen to know @janfreyberg?

**Comment by tknopp** [comment]
https://www.ncbi.nlm.nih.gov/pmc/pub/addjournal/

**Comment by janfreyberg** [comment]
So it looks like the initial application is relatively small, with more effort required during later steps (scientific quality review, and technical review).

The application is supposed to be handled by the publisher. Is there an editor on the JOSS team with remit for biomedical work who has the bandwidth to submit the applications? I'd be keen to help support this but am not an editor. Looking at the editorial board, maybe @Kevin-Mattheus-Moerman would be the right person?

**Comment by janfreyberg** [comment]
Hello, I just wanted to check if anyone knew who could drive this forward?

I also wanted to tag in some people who might be interested in the same - @emdupre, @mikldk @oliviaguest @chartgerink @prashjha all seem like editors who might be interested in seeing papers PubMed indexed.

Again, very happy to contribute and support this application as much as possible.

**Comment by mikldk** [comment]
I would be willing to help on this, yes.

**Comment by chartgerink** [comment]
I'd also be happy to support this effort in my role as an editor - I figure we should form a working group to go through all the steps in the application prior to applying, given that there is only one application every 24 months.

**Comment by janfreyberg** [comment]
Thanks both, that's amazing news!

Great suggestion @chartgerink - we can either collect the responses in this issue or start a new one?

For the initial application, I've just opened up the web form. Here are the questions & answers as far as I have them:

Page 1:

- Title
  - Journal of Open Source Software
- Sponsoring organization name
  - NumFOCUS
- Sponsoring organization URL
  - http://numfocus.org
- Country/Territory
  - United States
- Publication start year
  - 2016
- Publication frequency
  - Other
- Estimated number of peer-reviewed articles per year
  - 250
- In what format is the journal published?
- ISSN (electronic)
  - 2475-9066
- Is the entire journal open access?
  - Yes
- Are all articles published fully in English?
  - Yes
- Journal website URL
  - https://joss.theoj.org/
- Aims and scope (URL)
  - https://joss.theoj.org/about#submitting
- Names and Affiliations of the Editorial Board (URL)
  - https://joss.theoj.org/about#editorial_board
- Editorial policies (URL)
  - https://joss.readthedocs.io/en/latest/editing.html#expectations-on-joss-editors
- Peer Review Process (URL)
  - https://joss.readthedocs.io/en/latest/submitting.html#the-review-process
- Policies on Conflict of Interest, Human and Animal rights, and Informed Consent (URL)
  - https://joss.readthedocs.io/en/latest/reviewer_guidelines.html#joss-conflict-of-interest-policy
  - _AFAIK, there is no policy on Human/Animal rights or Informed Consent. I think this should be
    added given that some biomedical software packages do provide data for examples._
  - **(NB: this isn't a required field in the application)**
- Data sharing policy (URL)
  - _As above, currently there is none, but one should (IMO) be added_
  - **(NB: this isn't a required field in the application)**

Page 2

- Publisher Name
  - Open Journals
- Organization Address
  - _?? (@arfon)_
- Publisher website (URL)
  - http://www.theoj.org
- Business Structure
  - Non-profit organization
- Names and titles of organization owner and executives
  - _?? (@arfon)_
- Parent or related companies, and/or subsidiary organizations associated with the organization (include the names, URLs and relationship to applicant)
  - None ??
- Enter 'none' if your publishing company has no parent companies, related companies, and/or subsidiaries
- How do you select editors for your journal(s)?
  - ??
- Management team members
  - I assume this would be the steering council? https://github.com/openjournals/governance/blob/master/people.md
- Provide a summary for each member of the publisher's senior management team (e.g., executive editor, publishing director, managing editor, ethics manager, etc.).
  - _NB: I can open issues for each individual management team member, but some may prefer to disclose these privately?_
  - Name and title:
  - Responsibilities:
  - summary of responsibilities
  - Prior experience in scientific publishing:
  - Membership in professional associations:
- Publisher Policies
  - _These are all required in the web form. We could just link to the JOSS ones here, with a as well as a link to the governance document for Open Journals which states that policies are delegated to journals?_
  - Editorial policies (URL)
    - https://github.com/openjournals/governance/blob/master/governance.md#editorial-policies-and-processes
  - Peer review policy (URL)
    - N/A
  - Peer review policy (URL)
    - N/A
  - Advertising policy (URL)
    - N/A
  - Research ethics policy (URL)
    - N/A
  - Informed consent policy (URL)
    - N/A
  - The process for handling cases requiring corrections, retractions, and editorial expressions of concern (URL)
    - N/A

**Comment by Kevin-Mattheus-Moerman** [comment]
I have very little bandwidth at the moment to help (perhaps things improve over summer) but would be happy to help answer questions and be a relay point between yourself and the rest of the AEiC team.

**Comment by chartgerink** [comment]
Thanks a bunch @janfreyberg! 

I created [a Google Doc](https://docs.google.com/document/d/1q9XyfmrmpKWwu3h3AQsykxxaYqLbM6eE1ZQbixXv2xA/edit?usp=sharing) for us to start working through steps 1-6 (the application is only step 1). 

We can probably uncover some hurdles in later steps that need to be resolved before applying.

**Comment by vnmabus** [comment]
> We submitted to Web of Science a few weeks ago but haven't heard anything back yet.

Any update on this?

**Comment by chartgerink** [comment]
I apologize for not picking this up sooner - I am a bit overcommitted and have not been too structured about my JOSS time. I have now blocked 1h each week to work on JOSS related activities and will use that to also work on this! 😊

**Comment by chartgerink** [comment]
Hi @paloha - fwiw I am the one pushing PMC indexing along and we're waiting on finalizing one or two things in the JATS/XML before we can submit our application. I hope to be able to do that this year still.

**Comment by paloha** [comment]
@chartgerink thanks for quick reply. Do I understand it right, that finalizing JATS/XML will lead to submission of an application to PMC? Does this also somehow relate to indexing in Scopus and/or WoS?

**Comment by chartgerink** [comment]
Yes it is the final step for our submission application to PMC! 🥳 

I am not 100% about the relation to Scopus/WoS - I responded as this thread is about PMC. I figure it doesn't hurt 😉 If you're specifically interested about Scopus/WoS only it might be good to start a new thread.

**Comment by arfon** [comment]
>  I am the one pushing PMC indexing along and we're waiting on finalizing one or two things in the JATS/XML before we can submit our application. I hope to be able to do that this year still.

@chartgerink – can you remind me what we're waiting at this point? I know that @tarleb has (I think) addressed all of the issues with the JATS XML at this point.

> Hello @arfon & @tknopp, are there any news on the Scopus rebuttal?

@paloha – we're actually eligible to apply to Scopus again now (since more than 2 years has passed). I will endeavour to do this before the end of the year.

**Comment by paloha** [comment]
Hello @arfon, thanks for the update. Shall I still create a new issue as @chartgerink suggested? Being indexed in Scopus & WoS is a game changer because some institutions measure the performance of their researchers by the metrics of Scopus or WoS. So unfortunately, sometime a paper published does not "count" if not indexed in those.

**Comment by chartgerink** [comment]
@arfon - I've been out for a few weeks so it might've been fixed. The main issue that I remember was that we needed to change something about the images (last I checked it had relative path names only). I believe we needed to submit the images alongside the JATS/XML - I am getting back up to speed now after my vacation. Will try to figure this out this week.

**Comment by pydemull** [comment]
Hi all, I was wondering if there has been some news since the last message from this issue. I have recently published a paper in JOSS and I expected it would be added to my PubMed records list but when I see this discussion it seems that it is not yet something that is possible for "normal" JOSS papers (ie, without a particular funding as those from NIH for example). @samhforbes do you have any idea about what is currently possible to do to have a record on PubMed ? (maybe nothing for this moment ?).

**Comment by arfon** [comment]
@chartgerink @tarleb – I think we should have the images issue fixed now. @tarleb – can you confirm?

**Comment by samhforbes** [comment]
Hi @pydemull  it doesn't look like this is currently possible I'm afraid, although someone else might be able to correct me.

**Comment by tarleb** [comment]
@arfon: To the best of my knowledge, all image issues have been fixed.

The JATS file for each paper is in `joss.xxxxx/10.21105.joss.xxxxx.jats` (**not** `.xml`); images are placed in a `media` folder next to the JATS file. The image paths used in JATS files are relative the file, e.g., `media/results.png`.

**Comment by arfon** [comment]
⚡ thanks for confirming here @tarleb. @chartgerink – I think this addresses the issues we had here correct?

**Comment by chartgerink** [comment]
Yes indeed! 🚀 Thanks for the push on this issue @everyone 😊 

I am moving forward with the submission to PMC! Will post here when I'm done filling out the forms (doing it today).

![](https://media.giphy.com/media/Fo1cy8mqGDvbjpJBB7/giphy.gif?cid=ecf05e47et1xveddoi4eliq0a21afw0k7sv2niqyeosm3c46&rid=giphy.gif&ct=g)

**Comment by chartgerink** [comment]
Of course an error happens now:

![Screenshot 2022-12-15 at 13 38 32](https://user-images.githubusercontent.com/2946344/207861332-dd51fce5-be07-4e8b-8bdb-14e1c8c62126.png)

Emailing...

**Comment by chartgerink** [comment]
PMC hid a few questions in the form, pending a few final pieces of input I can submit this. I'll post here when I've done that. 🙌

**Comment by drcandacemakedamoore** [comment]
Any progress on this issue @chartgerink ? Is there anything we can do to help from the outside (I'm a potential submitter of more papers who hopes this will be Pubmed indexed ASAP)?

**Comment by chartgerink** [comment]
Thanks @drcandacemakedamoore! We're waiting on one email thread being resolved and then the application will go out! 

I will post here when that's done. Sorry it's taking so long! Never expected it to take this long.

**Comment by vnmabus** [comment]
Any update on the Web of Science status? The Google Doc is no longer available.

**Comment by arfon** [comment]
> Any update on the Web of Science status? The Google Doc is no longer available.

It's over two years since we submitted to WoS and as far as I know, we never heard back. We're planning a resubmission this spring.

**Comment by chartgerink** [comment]
This issue is about PubMed Central - please use another thread for Web of Science. 

PMC is still ongoing. Google doc is no longer active because we moved on from that. Submission is imminent.

**Comment by chartgerink** [comment]
I just sent the application to PMC! 🥳 

✅ Step 1 of 6

**Comment by iskandr** [comment]
Did they get back to you at all?

**Comment by chartgerink** [comment]
@iskandr I will post here as we progress throughout. I have no information as of this point.

**Comment by arfon** [comment]
> It's over two years since we submitted to WoS and as far as I know, we never heard back. We're planning a resubmission this spring.

Just a brief update to say that we've just resubmitted JOSS to Web of Science.

**Comment by chartgerink** [comment]
Also a short update from PMC: We received additional questions that we need to clarify in the JOSS policies, which we're working on. Here's a thread of issues we need to resolve to continue the process, as these require updates to JOSS directly.

- [x] First, we noted that the journal has a clearly defined conflict of interest policy for reviewers and editors, however, we were unable to locate any information about the journal's policy on conflicts of interest as it relates to authors. Please provide a link to the description of this journal policy. 
  - Please find that through the following link - https://joss.readthedocs.io/en/latest/submitting.html#conflict-of-interest-policy-for-authors  
- [ ] Next, can you please clarify the journal’s policies on corrections and retractions? Specifically, does the journal publish separate notices of correction and/or retraction with their own unique citation information? When reviewing an application, NLM looks for conformance with guidelines and best practices for the handling of corrections, retractions, and expressions of concern (e.g. ICMJE's [Recommendations for the Conduct, Reporting, Editing, and Publication of Scholarly Work in Medical Journals](http://www.icmje.org/icmje-recommendations.pdf), and COPE’s [Guidelines for retracting articles](https://publicationethics.org/files/retraction-guidelines.pdf)).
  - We post a notice at the end of the review thread ([example](https://github.com/openjournals/joss-reviews/issues/1277#issuecomment-667670005)). We also update the paper to make it clear that it has been retracted, and link to the retraction notice ([example](https://joss.theoj.org/papers/c44313ada36f12eebbaff10eb0888071))
  - [ ] As described in more detail in PMC’s policies on[ corrections](https://www.ncbi.nlm.nih.gov/pmc/about/guidelines/#corr) and[ retractions](https://www.ncbi.nlm.nih.gov/pmc/about/guidelines/#retr), PMC requires the submission of separate notices of correction or retraction, which must be labeled and published in citable form; that is, the correction or retraction should be readily discernable in the table of contents (or archive) and must be associated with identifiable e-location information (e.g. article ID). Once deposited, PMC creates links between corrected and retracted articles to their corresponding correction or retraction notice ([example](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3339402/)). These practices are in line with the best practices linked in my previous email. JOSS’ current method of adding a comment to the original peer review history thread in Github would not meet PMC’s requirements. Would JOSS consider updating its correction and retraction practices going forward to be in line with best practices?
- [ ] Lastly, we were unable to locate a table of contents or some other option to view journal content by publication year. During the scientific quality review for PMC, the reviewers generally look at published articles from the last two years of content. The ability to locate and access older articles or articles from a particular time period, whether through a Table of Contents/archive or through date/volume/issue filters, is consistent with the recommendations of[ PIE-J: The Presentation & Identification of E-Journals](https://www.niso.org/publications/rp-16-2013-pie-j) (NISO RP-16-2013, see section 2.6 Access to Content). Please let us know if earlier journal content can be accessed some other way besides clicking to the ‘Next’ page in the list of article at https://joss.theoj.org/papers/published.
  - We allow for filtering of year as follows
    https://joss.theoj.org/papers/year/2016
    https://joss.theoj.org/papers/year/2017
    ...etc
  - [ ] Thank you for providing the direct URLs, however, is there any way for users/readers to navigate to these pages from the journal website, or are these pages only accessible if a user knows the direct URL? Additionally, I noticed that the pages where content is filtered by year appear to display all papers (i.e. also includes pending/non-peer-reviewed papers). When I click the filter for ‘Published papers’ however, it goes back to displaying all papers regardless of year. Is there any sort of table of contents for published/accepted content to facilitate browsing by publication date, volume, or issue?
- [ ] How long can something stay in [pending](https://joss.theoj.org/papers/active)? Do articles ever get the boot from the pending stage?
- [ ] Does the journal have any update policy? Since code bases and technology update overtime, is there an expectation for the articles to be living documents?

**Comment by arfon** [comment]
> As described in more detail in PMC’s policies on corrections and retractions, PMC requires the submission of separate notices of correction or retraction, which must be labeled and published in citable form; that is, the correction or retraction should be readily discernable in the table of contents (or archive) and must be associated with identifiable e-location information (e.g. article ID). Once deposited, PMC creates links between corrected and retracted articles to their corresponding correction or retraction notice (example). These practices are in line with the best practices linked in my previous email. JOSS’ current method of adding a comment to the original peer review history thread in Github would not meet PMC’s requirements. Would JOSS consider updating its correction and retraction practices going forward to be in line with best practices?

@xuanxu – can we look at updating our handling of retractions? I think we should do something like this:

On a submission being retracted it should be possible for an EiC to run a follow-up command like `@editorialbot retract` and pass in a location to a retraction notice. The retraction notice should be a special kind of JOSS paper that doesn't have a software archive. I'm thinking something like this:

- EiC visits original review thread for the paper being retracted and types `@editorialbot retract --notice=https://github.com/openjournals/retraction-00001` :point_left: URL being where the retraction notice is stored.
- Editorialbot compiles the paper (pushing branch to `joss-papers`) and links to preview of paper to be retracted.
- If retraction notice looks OK, EiC does `@editorialbot retract --notice=https://github.com/openjournals/retraction-00001 --deposit=true` or something like that.
- Editorialbot publishes retraction notice.

Open questions:

- What should the DOI for the retraction notice be? I'm thinking it should be related to the original DOI (e.g., for `10.21105/00001` we could have `10.21105/00001R`)
- If the above works, what volume/issue/page number would we use (for volume and issue, I think we just use the normal scheme, but what about page number?)
- What should we do with the original paper? We might want to link it 'forward' to the retraction notice (e.g., similar to https://joss.theoj.org/papers/10.21105/joss.01277) but I'm not completely sure. Let's make sure we follow COPE's guidelines here: https://publicationethics.org/files/retraction-guidelines.pdf

>  Thank you for providing the direct URLs, however, is there any way for users/readers to navigate to these pages from the journal website, or are these pages only accessible if a user knows the direct URL? Additionally, I noticed that the pages where content is filtered by year appear to display all papers (i.e. also includes pending/non-peer-reviewed papers). When I click the filter for ‘Published papers’ however, it goes back to displaying all papers regardless of year. Is there any sort of table of contents for published/accepted content to facilitate browsing by publication date, volume, or issue?

@xuanxu – can we make some simple views (e.g., lists of linkable titles) that show on a single page all of the papers by issue/volume/year? The existing Rails filters should do most of the work here, but I think PMC is asking for the tables of contents to be easier to discover/browse (e.g., from the navigation menu).

> How long can something stay in pending? Do articles ever get the boot from the pending stage?

@chartgerink – I'm not sure I understand the question. Are they asking if articles can be booted out before they go for review? If so, the answer is 'yes'. In fact, about 25-30% of articles stop at this point (see stats in https://blog.joss.theoj.org/2023/05/JOSS-publishes-2000th-paper). This is mostly because of scope reviews and feedback to authors that their submission isn't ready to move forward.

> Does the journal have any update policy? Since code bases and technology update overtime, is there an expectation for the articles to be living documents?

We do support follow-up submissions for already-published software, provided the updates meet the same scholarly effort criteria as required in our submission guidelines (https://joss.readthedocs.io/en/latest/submitting.html#submission-requirements). I think this used to be documented somewhere but I can't find it now. 

Do you think I should write a short paragraph about this @chartgerink so we can provide an explicit link?

**Comment by xuanxu** [comment]
> @xuanxu – can we make some simple views (e.g., lists of linkable titles) that show on a single page all of the papers by issue/volume/year? The existing Rails filters should do most of the work here, but I think PMC is asking for the tables of contents to be easier to discover/browse (e.g., from the navigation menu).

Sure, I'll draft something

**Comment by xuanxu** [comment]
> * What should we do with the original paper? We might want to link it 'forward' to the retraction notice (e.g., similar to https://joss.theoj.org/papers/10.21105/joss.01277) but I'm not completely sure.

We could add a link to the sidebar on the paper page

![retractionlink](https://github.com/openjournals/joss/assets/6528/b6cb028c-d5ad-4f8c-b0b2-fe194bc90759)

**Comment by zchandler** [comment]
Fan of JOSS here. Is there anything that your supporters can do to help with this? Subscribing.

**Comment by chartgerink** [comment]
> @chartgerink – I'm not sure I understand the question. Are they asking if articles can be booted out before they go for review? If so, the answer is 'yes'. In fact, about 25-30% of articles stop at this point (see stats in https://blog.joss.theoj.org/2023/05/JOSS-publishes-2000th-paper). This is mostly because of scope reviews and feedback to authors that their submission isn't ready to move forward.

@arfon - I think you're correct. Maybe we should document this somewhere as well?

> Fan of JOSS here. Is there anything that your supporters can do to help with this? Subscribing.

@zchandler - at this time you cannot support the efforts as we're all in the hands of PubMed Central. We need to resolve the questions pending in https://github.com/openjournals/joss/issues/153#issuecomment-1571562457 but they're primarily things we editors need to fix. Thanks for offering! 😊

**Comment by xuanxu** [comment]
> * Thank you for providing the direct URLs, however, is there any way for users/readers to navigate to these pages from the journal website, or are these pages only accessible if a user knows the direct URL? Additionally, I noticed that the pages where content is filtered by year appear to display all papers (i.e. also includes pending/non-peer-reviewed papers). When I click the filter for ‘Published papers’ however, it goes back to displaying all papers regardless of year. Is there any sort of table of contents for published/accepted content to facilitate browsing by publication date, volume, or issue?

We have now a _Table of Contents_ page where papers can be browser by year/volume/issue: https://joss.theoj.org/toc
It is linked from every page in the footer.

**Comment by chadagreene** [comment]
> > It's over two years since we submitted to WoS and as far as I know, we never heard back. We're planning a resubmission this spring.
> 
> Just a brief update to say that we've just resubmitted JOSS to Web of Science.

Any updates on Web of Science? It appears that JOSS is still not listed in their [master list](https://mjl.clarivate.com/search-results), which could become an issue for me personally, as my employer only considers WoS-indexed publications in our annual salary review process.

**Comment by chartgerink** [comment]
Just documenting an update from our updated submission to PMC, dated Nov 8th, 2023:

> NLM has resumed reviewing the application for JOSS, now that the requested information has been provided. If any additional questions come up during the evaluation, we will let you know. Otherwise, it may take up to 16 weeks to complete the review, at which time you will be notified of the results via email.

**Comment by chartgerink** [comment]
@iskandr still awaiting their response - it is due end of month and I'll post back once I know more 😊

**Comment by chartgerink** [comment]
Hi all - I am relaying a response from Pubmed that I received fifteen minutes ago:

> We are pleased to inform you that Journal of Open Source Software has passed the Scientific Quality Review by NLM for PMC.

There are still next steps to realize the deposit, but we passed a big administrative hurdle! 🥳

**Comment by danielskatz** [comment]
Thanks @chartgerink for pushing this through!!

**Comment by arfon** [comment]
> There are still next steps to realize the deposit, but we passed a big administrative hurdle! 🥳

Woo hoo! 🎉

**Comment by alexlancaster** [comment]
> I noticed JOSS is listed on PubMed (https://www.ncbi.nlm.nih.gov/nlmcatalog/101708638) with
> 
> > In: PubMed: Selected citations only
> > Current Indexing Status: Not currently indexed for MEDLINE. Citations are for articles where the manuscript was deposited in PubMed Central (PMC) in compliance with public access policies. For further information, see Author Manuscripts in PMC.
> 
> I also noticed that there is exactly one JOSS paper indexed there. Is this because JOSS just got added, or what caused that paper to get indexed and not others? Will all future JOSS paper be indexed automatically?

I have a _JOSS_ manuscript in preparation, and I wanted to see if it was indexed in PubMed and did a quick search for the journal in PubMed a few months ago: 

https://pubmed.ncbi.nlm.nih.gov/?term=j+Open+Source+Softw%5Bjournal%5D

I only glanced it and saw a longish list of papers (I didn't actually check the number), so I assumed it was already indexed, so thought - great, it's indexed in PubMed - I'll submit to _JOSS_. 

Recently, I was a looking at the list bit more carefully and then I noticed that there only 68 papers in that list, which seemed too few given the total number on the journal website (but definitely up since this 2017 thread!).  I took a look and saw all of them were author manuscripts submitted in PMC (mainly from govt. agencies or HHMS) - not deposited directly by the journal.  This lead me to this thread.  So I'm happy that it will be more automatically eventually!

In the meantime, since I think a lot of people will be put off by non-automatic indexing in PubMed, they should know that as a temporary workaround, one way to get a _JOSS_ paper indexed in PubMed is to submit it directly as an author manuscript via the "NIH Manuscript Submission System": https://www.nihms.nih.gov/login/?next=/submission/ .  That's what I intend to do, if the direct deposition isn't up and running by then.  I have submitted author manuscripts before and it normally works great.

Meanwhile - I'm very happy that _JOSS_ is on this path to being in PubMed automaticaly! And I'm be happy to help in testing the deposition workflow if you need it. Diamond open-access is the future!

**Comment by alexlancaster** [comment]
> In the meantime, since I think a lot of people will be put off by non-automatic indexing in PubMed, they should know that as a temporary workaround, one way to get a _JOSS_ paper indexed in PubMed is to submit it directly as an author manuscript via the "NIH Manuscript Submission System": https://www.nihms.nih.gov/login/?next=/submission/ .  That's what I intend to do, if the direct deposition isn't up and running by then.  I have submitted author manuscripts before and it normally works great.

Does anyone know what the current status of automatic deposition in PMC is? I just got a paper accepted in _JOSS_, so I'm probably going to submit it via NIHMS as I described above, since I'm not seeing new papers being added yet. But if it's about to start then I can hold off (I assume if and when automation deposition starts it will detect existing entries and not create see duplicates?)

**Comment by alexlancaster** [comment]
> > In the meantime, since I think a lot of people will be put off by non-automatic indexing in PubMed, they should know that as a temporary workaround, one way to get a _JOSS_ paper indexed in PubMed is to submit it directly as an author manuscript via the "NIH Manuscript Submission System": https://www.nihms.nih.gov/login/?next=/submission/ .  That's what I intend to do, if the direct deposition isn't up and running by then.  I have submitted author manuscripts before and it normally works great.
> 
> Does anyone know what the current status of automatic deposition in PMC is? I just got a paper accepted in _JOSS_, so I'm probably going to submit it via NIHMS as I described above, since I'm not seeing new papers being added yet. But if it's about to start then I can hold off (I assume if and when automation deposition starts it will detect existing entries and not create see duplicates?)

Argh! I spoke too soon. I started the NIHMS submission and got all the way to the end, and then got this:

![image](https://github.com/user-attachments/assets/2f96135a-bb58-4abf-8cf5-be7bb1fa389a)

In our case, there are no federal grants to add, so it appear that this route to deposition PMC is not available unless there is a federal grant. So that leaves out work that is self-funded, even if it's peer-reviewed and in a journal that is approved to submit. I have written to NLM to get clarification. In the meantime that stinks, since it seems to cut off any direct submission work that although published in a peer-reviewed journal, if the journal isn't already in PubMed and also isn't funded off a federal grant, then it won't be indexed/deposited. That could represent a sizeable scientific work that is being missed.

**Comment by drcandacemakedamoore** [comment]
Yesm, I had a similar problem because my work was funded by a Dutch
institution. Putting in your own work is only possible on some limited
number of grants in the USA only.

**Comment by sneakers-the-rat** [comment]
Not finding the checklist that @chartgerink posted in the slack on here, i'll delete if it's somewhere i'm not seeing:

---

Specifically, can we set up a repository that mirrors joss-papers but is adjusted to meet the PMC criteria? For example, a joss-paper-pmc  repo that follows [the naming and delivery specs](https://www.ncbi.nlm.nih.gov/pmc/pub/filespec-delivery/) from PMC directly.

The primary differences are that we need to do some renaming/reorganizing per paper. The specific action items I would see based on my evaluation are:

- [ ] Each paper has its own folder named `joss-vol-uid` compared to what is now `joss.uid`
- [ ] The paper files follow the same format (`joss-vol-uid.xml` and `joss-vol-uid.pdf`)
- [ ] All media files live in the root folder for a paper, and are named `joss-vol-uid-g001.ext` (so no more subfolders; this also requires the XML file refs to be changed)
- [ ] Convert images from SVG to [approved formats](https://www.ncbi.nlm.nih.gov/pmc/pub/filespec-images/)

---

Also @tarleb mentioned some issues in https://github.com/openjournals/inara and I see some closed but not sure the status of that.

**Comment by chartgerink** [comment]
@alexlancaster we're still working on getting everything up to PMC's specs - they are particular in their requirements so it is taking a moment. 

> [!NOTE]
> As a general rule: If I don't post updates on PMC's correspondence, there are no updates. 

Thanks @sneakers-the-rat for the copy over here!

**Comment by sneakers-the-rat** [comment]
I've gotten started on this over here: https://github.com/sneakers-the-rat/joss-papers-pmc
i guess there is disagreement about whether we will do all this in-place in the `joss-papers` repo or in a parallel one (i prefer the latter) but in either case should be able to do either once we have the migration code in place.

it turns out that it's not so trivial as we would have hoped since there have been a number of errors so far from inconsistent/incorrect metadata, i'm raising those as I go over in https://github.com/openjournals/joss-papers

Since I'm assuming we'll want this to be auditable, i've started making a reproducible series of steps over here so ppl can check my work (and also if you want to contribute then that's np i can make you a collaborator on the repo) along with some logs as i do the steps: https://github.com/sneakers-the-rat/joss-migrate-pmc

**Comment by alexlancaster** [comment]
> Since I'm assuming we'll want this to be auditable, i've started making a reproducible series of steps over here so ppl can check my work (and also if you want to contribute then that's np i can make you a collaborator on the repo) along with some logs as i do the steps: https://github.com/sneakers-the-rat/joss-migrate-pmc

Happy to help with testing. I'll have some time starting in September.

**Comment by yarikoptic** [comment]
ha, @sneakers-the-rat our ways crossed again -- happy to see recent activity to get automated submissions from JOSS to PMC, since now a colleague demands me to upload our recent paper there MANUALLY! ;)

**Comment by aloctavodia** [comment]
Any update on this?

**Comment by chartgerink** [comment]
For complete disclosure: I handed this over to @arfon, who is the best person (as far as I know) to provide an update 👍
